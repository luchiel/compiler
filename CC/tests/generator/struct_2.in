struct point { int x, y; };

void a(struct point c)
{
c.x = c.x + 4;
printf("%d %d\t", c.x, c.y);
}

void main()
{
struct point m;

m.x = 6;
m.y = 7;

a(m);

printf("%d %d\t", m.x, m.y);
}