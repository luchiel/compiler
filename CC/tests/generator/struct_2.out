10 7	6 7	